DevIncubator_Autopark/Program.cs

```diff
@@ -4,9 +4,44 @@ namespace DevIncubator_Autopark
 {
 	class Program
 	{
-		static void Main(string[] args)
+		static void Main()
 		{
-			Console.WriteLine("Hello World!");
+			//1
+			VehicleType[] vehicles = new VehicleType[]
+			{
+				new VehicleType("Bus", 1.2f),
+				new VehicleType("Car"),
+				new VehicleType("Rink", 1.5f),
+				new VehicleType("Tractor", 1.2f)
+			};
+
+			//2, 5, 6
+			float averageCoefficient = 0;
+			for (int i = 0; i < vehicles.Length; i++)
+			{
+				vehicles[i].Display();
+
+				averageCoefficient += vehicles[i].TaxCoefficient;
+			}
+
+			averageCoefficient /= vehicles.Length;
+			Console.WriteLine($"Average tax coefficient = {averageCoefficient:0.00}");
```

Comment:
Create pls separate methods for calculation of avarage and max values. It's more readable. As I see these methods will accept collection of vihicles and return floating point number.

Reply:
fixed